src/dashboard/cd-dashboard-news.vue

```diff
@@ -170,5 +175,41 @@
       }
     }
 
+    &__cta {
+      text-align: center;
+    }
+
+    &__view-all {
+      color: @cd-purple;
+      border: 1px solid @cd-purple;
+      font-size: 16px;
```

Comment:
What's our default font-size? Feel free to define a new variable for that, like font-size-cta if it's missing

Reply:
for buttons? 16px as i can see from others

Reply:
It's not because it's not properly done that you should do the same :)
What's the value for @font-size-medium ? If it doesn't fit, define a new variable like "font-size-button" or whatever seems adequate in variables.less :)